pyroclast/common/tf_util.py

```diff
@@ -283,3 +283,30 @@ def finalize(self):
             self.sum_first)) * tf.math.sqrt(self.n * self.sum_sq_second -
                                             tf.math.square(self.sum_second))
         return numerator / (denominator + 1e-12)
+
+
+def load_model(module,
+               model_save_name,
+               data_dict,
+               output_dir=__file__ + '/../../../',
```

Comment:
This seems very specific...

Reply:
It sort of is. __file__ is very specific so we have to adjust to it. Currently, it points to the project root directory.